Client/Frontend/Menu/Items/MenuItemView.swift

```diff
@@ -0,0 +1,106 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+/**
+ * creates a view that consists of an image and a title.
+ * the image view is displayed at the center top of the
+ **/
+public class MenuItemView: UIControl {
```

Comment:
When I added this in my sample project using just iOS9, adding the custom views to a UIButton subclass caused the button not to display, whereas subclassing the UIControl did work. Just turned the UIControl into a UIButton here and everything layed out just fine, so I guess I was doing something wrong on the sample menu project. Updating.